frontend/src/metabase/visualizations/visualizations/IFrameViz/utils.ts

```diff
@@ -0,0 +1,117 @@
+import { isSafeUrl } from "metabase/lib/formatting/link";
+
+const embedLinkTransformers: {
+  test: (url: URL) => boolean;
+  transform: (url: URL) => string;
+}[] = [
+  {
+    test: url => ["loom.com", "www.loom.com"].includes(url.hostname),
```

Comment:
Still incorrect, e.g. `om.com` or `loom.com.org` will give false positives.
IMO it's best to handle these with regexps (which is what [the libs](https://github.com/metabase/metabase/pull/47994#issuecomment-2384860716) do).